include/linux/compiler.h

```diff
@@ -249,11 +249,11 @@ static __always_inline void __write_once_size(volatile void *p, void *res, int s
  */
 
 #ifdef CONFIG_KTSAN
-void ktsan_sync_acquire(void *addr);
-void ktsan_sync_release(void *addr);
+void ktsan_sync_nonmat_acquire(void *addr);
```

Comment:
we will choke with this new terminology, nonmat is unclear to me and Paul McKenney
C already has established, standardized terminology and APIs for all of this

nonmat_acquire is atomic_load(memory_order_relaxed)
bitop_clear_bit_unlock is atomic_fetch_and(memory_order_release)